app/components/RadioPlayer.vue

```diff
@@ -112,7 +112,7 @@
           </div>
         </div>
 
-        <div class="sand-clock">
+        <div v-if="false" class="sand-clock">
           <IconSandclock :progress="time_percent" :live="np.live.is_live" />
```

Comment:
Do we disable the sand clock completely?

Reply:
Csenge told me also she finds it unneeded and distracting

Reply:
Otherwise it is of course open to discussion on the next meeting

Reply:
Oh, that's a bold change :) I would say, let's wait until the meeting before we merge this change. And if we decide to get rid of the sand clock, I think it's better that we delete it from code (we have it in the history in case we want to bring it back some time in the future).